hw/ip/usbdev/data/usbdev_testplan.hjson

```diff
@@ -654,11 +654,11 @@
     {
       name: stall_trans
       desc: '''
-            Verify that by enabling in_stall/out_stall shall cause STALL response to
-            attempted IN/OUT transactions.
+            Verify that by enabling out_stall shall cause STALL response to
+            attempted OUT transactions.
 
-            - setting in_stall/out_stall register bits for a particular transaction.
-            - Verify that we will get STALL response when try to attempt IN/OUT transactions.
+            - Setting out_stall register bits for a particular transaction.
+            - Verify that we will get STALL response when try to attempt OUT transactions.
```

Comment:
More grammar (again): We need a subject for the verb :-) This should probably be "... when *we* try to ..."